### PR TITLE
feat: enable drivers for Rock64

### PR DESCRIPTION
**Pull request by andrewrynhard**
This enables a number of drivers.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

File: kernel/kernel/config-arm64 [hunk 1]
```diff
@@ -543,10 +545,10 @@ CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_GOV_ATTR_SET=y
 CONFIG_CPU_FREQ_GOV_COMMON=y
 CONFIG_CPU_FREQ_STAT=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+# CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE is not set
 # CONFIG_CPU_FREQ_DEFAULT_GOV_POWERSAVE is not set
 # CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE is not set
-# CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND is not set
+CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
```
Comment (smira) [inline kernel/kernel/config-arm64:551]: I believe this enables CPU frequency scaling by default, ... should we really do it?

Comment (andrewrynhard) [reply kernel/kernel/config-arm64:551]: We should probably drop the cpu stuff.

**Review by smira — APPROVED** [review]

**Review by rsmitty — APPROVED** [review]

**Pull request merged**